base/context.go

```diff
@@ -113,18 +113,14 @@ func (ctx *Context) GetClientTLSConfig() (*tls.Config, error) {
 	}
 
 	if ctx.Certs != "" {
-		if log.V(1) {
-			log.Infof("setting up TLS from certificates directory: %s", ctx.Certs)
-		}
+		log.Printf("setting up TLS from certificates directory: %s\n", ctx.Certs)
```

Comment:
we use `Context.User` and `Context.HTTPRequestScheme()` in addition to `GetHTTPClient`